CHANGES.md

```diff
@@ -1,6 +1,433 @@
 Release Changes List
 ====================
 
+12345678901234567890123456789012345678901234567890123456789012345678901234567890
+
+version 1.18.0
+==============
+
+Twenty-first public release of Chapel, September 20, 2018
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+
+File / Directory Structure
+--------------------------
+* added CHPL_HWLOC and CHPL_TARGET_MEM to qthread-specific paths
+* added JEMALLOC_LIB_DIR to the runtime path
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* records and classes now use initializers by default
+* deprecated constructors in favor of initializers
+* deprecated support for extern classes
+* for a class C, 'new C(...)' now is equivalent to 'new borrowed C(...)'
+  (see TODO)
+* deprecated 'Owned(C)' and 'Shared(C)' in favor of 'owned C' and 'shared C'
+* made forall loops borrow outer owned and shared class objects by default
+* the default intent for formals declared with owned/shared types is 'const in'
+* when overriding a class method, the 'override' keyword must now be used
+  (see (see 'Overriding Base Class Methods' in the spec's 'Classes' chapter)
+* deprecated the interpretation of ':' within where clauses as a subtype query
+* deprecated support for casting from numeric types to 'c_string'
+
+New Features
+------------
+* added support for managed class objects via 'owned', 'shared', and 'borrowed'
+  (see TODO)
+* added a compiler lifetime checker that flags borrows referring to freed memory
+* added an 'unmanaged' keyword for manually managed class objects
+* added a `.borrow()` method to all flavors of classes
+* added support for array return types with unspecified domains / element types
+  (see TODO)
```

Comment:
I've changed my mind on the documentation link for this, since the one in the spec ended up being really simple.  Unless you think an example would be good here